docs/topics/plugin-install.adoc

```diff
@@ -20,3 +21,18 @@ NOTE: This guide uses _Eclipse_ to refer to an installation of Eclipse or JBoss
 . Review the installation details and press *Next*.
 . Accept the terms of the license agreement and press *Finish* to install the plugin.
 . Restart Eclipse for the changes to take effect.
+
+.Install the Plugin for an Offline Environment
+
+. Download the link:{ProductDownloadURL}{EclipsePluginFilename}-{ProductVersion}.zip[{ProductShortName} tooling].
+. Launch Eclipse
+. From the menu bar, select *Help* -> *Install New Software*.
+. Add the {ProductShortName} update site.
+.. Next to the *Work with* field, click *Add*.
+.. In the *Name* field, enter `{ProductShortName} - Offline`.
+.. Next to the *Location* field, click *Archive*.
+.. Select the file downloaded in the first step, click *Ok*.
```

Comment:
Suggest changing "in the first step, click *Ok*" to "in the first step and click "Ok".

Also, should "Ok" be "OK"? (I'm not currently looking at the GUI to know).

Reply:
I've adjusted to use your text, and you were correct - the GUI uses OK instead of Ok.